src/ducks/balance/index.js

```diff
@@ -1,4 +1,4 @@
-import Balance from 'ducks/balance/Balance.jsx'
+import Balance from 'ducks/balance/BalanceWithBanksJobs'
```

Comment:
```
import BalanceWithBanksJobs from 'ducks/balance/BalanceWithBanksJobs'
```

grep test is hard when the name is not the same.

Reply:
But BalanceWithBanksJobs is still greppable. And If modify the exported name to BalanceWithBanksJobs, I will have to modify the maif override accordingly, Do you think it is worth it ?